helm/values.yaml

```diff
@@ -80,9 +80,12 @@ autoscaling:
   targetCPUUtilizationPercentage: 40
   # targetMemoryUtilizationPercentage: 80
 
-podDisruptionBudget:
-  enabled: false
-  minAvailable: 1
+# Set createPodDisruptionBudget to true and uncomment
+# podDisruptionBudget block below to enable pod disruption budget
+createPodDisruptionBudget: false
+
+# podDisruptionBudget:
+#   minAvailable: 1
```

Comment:
Let's change this to the following instead of introducing a new value

```
podDisruptionBudget
#  minAvailable: 1
```

Reply:
Did you mean it to be 
`podDisruptionBudget:` or `podDisruptionBudget`